projects/js-packages/components/components/threat-modal/styles.module.scss

```diff
@@ -10,6 +10,24 @@
 	gap: calc( var( --spacing-base ) * 2 ); // 16px
 }
 
+.section .section__toggle {
+	text-decoration: none;
+
+	&:focus {
+		box-shadow: none;
+	}
+
+	&:hover {
+		text-decoration: underline;
+	}
+
+	&_content {
```

Comment:
Nit/opinion - better to use full class names so they can be searched, i.e. "section__toggle_content".

If "content" is a separate block, a double `__` might be wanted here.